pkg/kubectl/cmd/util/editor/editoptions.go

```diff
@@ -500,11 +506,7 @@ func getPrinter(format string) *editPrinterOptions {
 	}
 }
 
-func (o *EditOptions) visitToPatch(
-	originalInfos []*resource.Info,
-	patchVisitor resource.Visitor,
-	results *editResults,
-) error {
+func (o *EditOptions) visitToPatch(originalInfos []*resource.Info, patchVisitor resource.Visitor, results *editResults) error {
```

Comment:
I don't think we really split args onto separate lines anywhere, looked weird indeed.